lib/phlex/html/callbacks.rb

```diff
@@ -1,11 +1,21 @@
 # frozen_string_literal: true
 
 module Phlex::HTML::Callbacks
-	def template(&block)
-		before_rendering_template if respond_to?(:before_rendering_template)
+	def self.prepended(mod)
+		raise "Phlex::HTML::Callbacks should be included rather than prepended."
+	end
```

Comment:
Temporary "deprecation" warning in case anyone is prepending `Phlex::Callbacks`.